tests/test_series.py

```diff
@@ -3352,3 +3353,20 @@ def test_vector_series_normalize():
         [(x, -5, 5), (y, -5, 5), (z, -5, 5)], params={u: 1},
         slice=Plane((0, 0, 0), (0, 1, 0)), normalize=True)
     assert s.normalize
+
+def test_complex_number_eval():
+    xi, wn, x0, v0, t = symbols("xi, omega_n, x0, v0, t")
+    x = Function("x")(t)
+    eq = x.diff(t, 2) + 2 * xi * wn * x.diff(t) + wn**2 * x
+    sol = dsolve(eq, x, ics={x.subs(t, 0): x0, x.diff(t).subs(t, 0): v0})
+    d = {
+        'params' : {
+            wn: 0.5,
+            xi: 0.25,
+            x0: 0.45,
+            v0: 0.0}
+    }
+    s = LineInteractiveSeries([sol.rhs], [(t, 0, 100)], n1=5,**d)
+    x_axis, y_axis = s.get_data()
+    assert np.all([x != np.nan for x in x_axis])
+    assert np.all([y != np.nan for y in y_axis])
```

Comment:
These assertions are wrong. With Numpy we have to use `np.isnan` to test for nans. To convince yourself, you can try this:

```py
a = np.nan
print(a == np.nan) # false
print(np.isnan(a)) # true
```

Reply:
sorry, i will fix it.